clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -941,6 +941,19 @@ struct CounterCoverageMappingBuilder
     return Counter::getCounter(CounterMap[S]);
   }
 
+  std::pair<Counter, Counter> getBranchCounterPair(const Stmt *S,
+                                                   Counter ParentCnt) {
+    Counter ExecCnt = getRegionCounter(S);
+    return {ExecCnt, Builder.subtract(ParentCnt, ExecCnt)};
+  }
+
+  bool IsCounterEqual(Counter OutCount, Counter ParentCount) {
```

Comment:
Should OutCount and ParentCount be const?

Reply:
Does non-const byval arg prevent `const`? I'll take a look later.

Reply:
Did you try this?

Reply:
No. I didn't think it valuable. Could you elaborate why adding `const` is useful here?